ckanext/harvest/commands/harvester.py

```diff
@@ -20,6 +20,9 @@ class Harvester(CkanCommand):
       harvester rmsource {id}
         - remove (inactivate) a harvester source
```

Comment:
This could be improved to be: "remove (deactivate) a harvester source, whilst leaving any related datasets, jobs and objects."